### ISSUE TITLE
Zenodo -- 403 on review page

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2954]
When curators try to view Zenodo content from the submission system review page, they get a 403 error "most of the time". 

(This may be covered by one of the other Zenodo issues, but it's not obvious...)

**Comment by bryanmgee** [comment]
FYI this is still happening for PPR link Zenodo files as well (example that author just emailed about: https://datadryad.org/stash/share/AnRlHWxaKj-Uhn9T_53eDJAozpTR_G2O1o1fvreKbEk)

**Comment by bryanmgee** [comment]
This is an example where I tried on the Review and Submit page: https://datadryad.org/stash/ds_admin/activity_log/119574

**Comment by sfisher** [comment]
I wonder if Zenodo has changed something about our key.  I'll find some others and see if they're all still failing.  It's definitely a permissions error since it works fine in my main browser where I'm already logged into Zenodo but fails in incognito mode where I'm not.

I'll recheck our config (to be sure we're using the one true magic key) and that it seems to be failing everywhere and email them about it again.

**Comment by sfisher** [comment]
I've checked our config for changes and it seems everything is fine and the same it was when it was working recently.

@ryscher To answer the question you asked in standup. Yes, this affects both the "private for peer review" page and also the review page in the last step of the submission process.  They both use RAT urls for Zenodo and both fail.  I created a new dataset to test this.  I believe this is happening for all datasets.  To be sure you see it failing be sure you're logged out of Zenodo in your browser (it will work if you're logged into the Zenodo account that has access, but a user would never have this account or this access).

I'll put in an email to the Zenodo people to let them know we've recently seen it breaking and I suspect they need to add something special back to our access token that was removed.

**Comment by sfisher** [comment]
Emailed Zenodo with information on the problem on 2023-11-16 .

**Comment by sfisher** [comment]
This was fixed for a short time (like a week) but it has come back again.  I've emailed them about it again.  Things work if you're logged into our account in Zenodo, so it's best to use an incognito tab for testing this.

When it worked we had to 1) not use the OAuth limited time tokens and use the long-lived token they assigned to our account manually and 2) Zenodo had to update something special about our token in their system.

I verified we're still using the pre-assigned long-lived token and I don't believe our Remote Access Token code has changed at all and looking for a fix from them.
